### DISCUSSION TITLE
[DropdownMenu] Maintaining focus on a text input

### DISCUSSION THREAD

**Discussion by ThePaulMcBride** [radix-ui/primitives#1009] [Help]
I'm trying to use the drop down menu where one of the items is a simple text/number input and running into issues. Not sure if this is just flat out not supported or not, but it would be very useful!

Essentially, when the input is focused, if the mouse is moved, it will lose focus.

Any ideas on how I can stop that from happening?

Here is a minimal reproduction:

https://n0fzt.csb.app/

**Comment by jjenzz** [comment]
Hi @ThePaulMcBride ,

Unfortunately this would be a misuse of the [DropdownMenu pattern](https://w3c.github.io/aria-practices/examples/menu-button/menu-button-actions.html). The menu items have `role=menuitem` (so SR users can interpret them as clickable actions) and various other accessibility/functionalities applied specific to that pattern so if you try to break away from that you will likely end up with something that isn't accessible.

What type of component are you trying to create here?

**Reply by ThePaulMcBride** [reply]
Hey @jjenzz!

Here are the designs of what I'm trying to build. If this is likely to end up being inaccessible, do you have any ideas as to what I should do instead?

<img width="647" alt="Screenshot 2021-12-02 at 13 39 24" src="https://user-images.githubusercontent.com/6891650/144432901-1ec75b14-d84b-43d8-a56e-23916a6d4294.png">

**Reply by jjenzz** [reply]
Is this portion opened via a button or is it always on screen? Would you mind sharing the whole screenshot 🙏

![image](https://user-images.githubusercontent.com/175330/144615141-d544a039-01b2-41a0-ba90-1b5f840cb143.png)

**Comment by gabrielpetersson** [comment]
For those of you who need an input bar and are confused about why input loses focus, switch out <DropdownMenu.Item> for a div (might not be accessible, but I'm building an internal tool so that's ok)
